src/js/media-chrome-range.ts

```diff
@@ -121,6 +121,7 @@ template.innerHTML = /*html*/ `
 
     #appearance {
       height: var(--media-range-track-height, 4px);
+      transition: var(--media-range-track-transition);
```

Comment:
this was functionality was removed with the re-architecture. 
just making it work again here.

it's possible to animate the track via height but it's not so smooth as doing it with a transform.